zettarepl/transport/zfscli/__init__.py

```diff
@@ -78,6 +79,17 @@ def get_receive_resume_token(shell, dataset):
     return get_property(shell, dataset, "receive_resume_token")
 
 
+def get_all_properties_raw(shell, dataset):
+    with ZfsCliExceptionHandler():
+        output = shell.exec(["zfs", "get", "-H", "-o", "property,value", "all", dataset])
```

Comment:
isnt this an expensive call, especially for a lot of datasets?

I wonder if we can just fix by excluding the mountpoint from VOLUME type of datasets?